src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,39 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const { user, body, title } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <a className="UserInfo" href={user?.email}>
+            {user && <UserInfo user={user} />}
+          </a>
```

Comment:
Move tag `a` inside UserInfo component
```suggestion
        {user && <UserInfo user={user} />}
```

![Uploading image.png…]()